File: backend/src/main/java/com/it/finance/personal_finance_be/dto/EconomicAccountDto.java
package com.it.finance.personal_finance_be.dto;

import com.it.finance.personal_finance_be.framework.PfTypoDto;

public class EconomicAccountDto extends PfTypoDto {

    private EconomicCategoryDto economicCategory;

    public EconomicCategoryDto getEconomicCategory() {
        return economicCategory;
    }

    public void setEconomicCategory(EconomicCategoryDto economicCategory) {
        this.economicCategory = economicCategory;
    }

}

File: frontend/src/dto/finance/EconomicAccountDto.ts
import PfTypoDto from "@/dto/framework/PfTypoDto.ts";
import type EconomicCategoryDto from "@/dto/finance/EconomicCategoryDto.ts";

export default class EconomicAccountDto extends PfTypoDto {

    economicCategory: EconomicCategoryDto | null = null;

}